includes/AppLeave.php

```diff
@@ -92,6 +92,8 @@
 	//
 	require "DetermineApplnPath.php";
 	
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->